src/sagas/form.js

```diff
@@ -0,0 +1,38 @@
+import {put, select, takeEvery } from 'redux-saga/effects';
+
+import {storeInputChange, cleanForm} from '../actions/form';
+import {ActionTypes} from '../common/constants';
+
+
+const getForm = state => state.form;
+
+export function* processFormChange(action) {
```

Comment:
You don't really need saga for form change. There is no async operation that you need to deal with here. As it is, you can deal with form change in reducer alone. The only thing I suggest to take care in saga regarding saving movie is saving it itself. Altho it is not necessary as we don't make any API request to do it.